.github/workflows/security-issue-label-checker.yml

```
@@ -0,0 +1,82 @@
+name: Check and Complete Team Labels
+
+on:
+  issues:
+    types: [opened, edited, labeled]
+
+permissions:
+  issues: write
+
+jobs:
+  ensure-team-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check and add missing labels
+        uses: actions/github-script@v7
```

```suggestion
        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
```

We pin these versions for security reasons and Renovate updates.